packages/shared/src/components/profile/UserList.tsx

```diff
@@ -53,7 +53,6 @@ function UserList({
               href={user.permalink}
               user={user}
               afterContent={afterContent?.(user, i)}
-              showFollow
```

Comment:
Now the default is false and I also enabled it on cases where we support following list user list modal and following filter settings.